engine-tests/src/tests/eth_connector.rs

```diff
@@ -518,14 +528,45 @@ fn test_ft_transfer_call_without_message() {
     );
     res.assert_success();
 
+    // some-test-acc does not implement `ft_on_transfer` therefore the call fails and the transfer is reverted.
     let balance = get_eth_on_near_balance(&master_account, DEPOSITED_RECIPIENT, CONTRACT_ACC);
     assert_eq!(balance, DEPOSITED_AMOUNT - DEPOSITED_FEE);
-
     let balance = get_eth_on_near_balance(&master_account, "some-test-acc", CONTRACT_ACC);
-    assert_eq!(balance, 2 * transfer_amount);
+    assert_eq!(balance, 0);
+    let balance = get_eth_on_near_balance(&master_account, CONTRACT_ACC, CONTRACT_ACC);
+    assert_eq!(balance, DEPOSITED_FEE);
 
+    // Sending to external receiver with empty message should be success
+    let ft_receiver = master_account.deploy(
+        &ft_receiver_bytes(),
+        "ft-rec".parse().unwrap(),
+        near_sdk_sim::STORAGE_AMOUNT,
+    );
+    let res = recipient_account.call(
+        CONTRACT_ACC.parse().unwrap(),
+        "ft_transfer_call",
+        json!({
+            "receiver_id": ft_receiver.account_id(),
+            "amount": transfer_amount.to_string(),
+            "msg": "",
+        })
+        .to_string()
+        .as_bytes(),
+        DEFAULT_GAS,
+        1,
+    );
+    res.assert_success();
+
+    let balance = get_eth_on_near_balance(&master_account, DEPOSITED_RECIPIENT, CONTRACT_ACC);
+    assert_eq!(balance, DEPOSITED_AMOUNT - DEPOSITED_FEE - transfer_amount);
+    let balance = get_eth_on_near_balance(
+        &master_account,
+        ft_receiver.account_id().as_ref(),
```

Comment:
```suggestion
        dummy_ft_receiver.account_id().as_ref(),
```

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/330/commits/a9c6a2908ce489696d9c7a6f7ac8a11572e3153f